test/es-module/test-typescript-commonjs.mjs

```diff
@@ -1,6 +1,6 @@
 import { skip, spawnPromisified } from '../common/index.mjs';
 import * as fixtures from '../common/fixtures.mjs';
-import { match, strictEqual } from 'node:assert';
+import * as assert from 'node:assert';
```

Comment:
This seems unnecessary and unrelated to the actual change

Reply:
I got it back thank you